src/Microsoft.Azure.Functions.Extensions.Mcp/Http/StreamableHttpTransport.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO.Pipelines;
+using Microsoft.Azure.Functions.Extensions.Mcp;
+using ModelContextProtocol.Protocol;
+using ModelContextProtocol.Server;
+
+internal sealed class StreamableHttpTransport(bool flowExecutionContext = false)
+    : McpExtensionTransport<StreamableHttpServerTransport>(new StreamableHttpServerTransport() { FlowExecutionContextFromRequests = flowExecutionContext })
+{
+    private Func<StreamableHttpTransport, InitializeRequestParams?, ValueTask>? _onInitRequestReceived;
+
+    public override Task HandleMessageAsync(JsonRpcMessage message, CancellationToken cancellationToken)
+    {
+        throw new NotSupportedException();
+    }
+
+    public override string? SessionId
+    {
+        get => Transport.SessionId;
+        set => Transport.SessionId = value;
+    }
+
+    public Task HandleGetRequest(Stream sseResponseStream, CancellationToken cancellationToken)
+        => Transport.HandleGetRequest(sseResponseStream, cancellationToken);
+
+    public Task<bool> HandlePostRequest(IDuplexPipe httpBodies, CancellationToken cancellationToken)
```

Comment:
Should both of these contain `Async` in the name?

Reply:
I'm assuming you're talking about the methods on `StreamableHttpTransport`. Those we can, but the transport methods come from the SDK. Those were simply proxied here (hence the name), but I can add the suffix.

Reply:
Yeah I meant `HandlePostRequest` and `HandlePostRequest` - just trying to be consistent with out async methods